packages/config/lib/options.ts

```diff
@@ -140,11 +140,6 @@ const resolvedOptions: Array<ResolvedConfigOption> = [
     defaultValue: 60000,
     validation: validate.isNumber,
     canUpdateDuringTestTime: true,
-  }, {
-    name: 'exit',
-    defaultValue: true,
-    validation: validate.isBoolean,
-    canUpdateDuringTestTime: false,
```

Comment:
Why does removing this fix the issue? :thinking: 

Do you mind explaining a little more about this fixed the regression (not entirely clear what part of the regression PR linked broke this in the first place).

Reply:
@lmiller1990 I've updated this PR description with more details about the regression. The expected behavior is that Cypress will pause if it is in open mode _or_ if it in run mode with `--headed` and `--no-exit` both passed as args. Therefore, if cypress is in run mode aka not interactive (`!config('isInteractive')`) **and** either `--headed` or `--no-exit` was not passed, then we **do not** want to pause. Hence, the logic in the snippet above

Reply:
Right, I see! I hope we can have `config('mode')` in the future - `config('mode') === 'run` would be way more readable. Nice job hunting down this regression.